### DISCUSSION TITLE
如何申请 Filecoin Next Step Micro-grants?

### DISCUSSION THREAD

**Discussion by longfeiWan9** [filecoin-project/community-china#125] [社区公告]
Filecoin Next Step Micro-grants 主要用于支持在尝试使用 Filecoin 创建初始原型后想要继续往下探索的项目或者个人。去中心化技术是一个快速发展的领域，存在许多未知数，所以Filecoin希望可以通过Next Step Microgrants 来支持符合这种快节奏、具有试验以及探索精神的项目。
> 更多关于Filecoin grants的信息，请参考 [Filecoin Dev Grants](https://grants.filecoin.io/)

##### 重点：

+ 用来支持想要在Filecoin做贡献的个人、小项目、非盈利、活动组织者以及研究员等。
+ 为了让更多开发者尝试使用FIlecoin, 审批时不考虑长久的计划和影响
+ 统一资助$5,000, 并且每周会统一审批并回复
+ 开发时间不超过3个月

##### 基本要求：

1. 已经尝试过开发Filecoin相关的内容；
2. 描述在micro-grant的资助下，你对该项目的下一步计划是什么；
3. 申请的项目需要在3个月内完成；
4. 项目需要开源，代码使用MIT license & 内容的话使用 [CC-BY-SA 3.0](https://ipfs.io/ipfs/QmVreNvKsQmQZ83T86cWSjPu2vR3yZHGPm5jnxFuunEB9u) license；
5. 需要在资助期间汇报每周的进度，并且在项目完成以后提供整体汇报；

##### 申请流程

1. 提交[Discussions > Project Showcase](https://github.com/filecoin-project/community/discussions/categories/project-showcase)

   在[Discussions > Project Showcase](https://github.com/filecoin-project/community/discussions/categories/project-showcase)下创建一个新的`Discussion`，`Category`选择`Project Showcase`, 标题为: `[Application] Your Project Name`. 目的是为了和社区同步项目计划、更新进展以及和接触潜在用户。参考 [[Applicaion] Miner.Report](https://github.com/filecoin-project/community/discussions/208).

2. 在[filecoin-project/community](https://github.com/filecoin-project/community/#showcase-your-project) 提交项目展示Pull Request

   + 根据 [project submission template](https://github.com/filecoin-project/community/blob/master/templates/project-submission-template.md) 创建你的项目的介绍文件，根据项目填写相关信息，并且以项目名命名文件,并放在`community/projects`下的相应的文件夹内。在"How the community can engage" 处，链接到上一步创建的Project Showcase。
   + 项目介绍文件完成以后，提交Pull Request，标题为: `[Project submission] Your Project Name`。参考：[Project showcase submission](https://github.com/filecoin-project/community/blob/master/projects/application/miner-report-en.md)
   + 切记在PR的描述中，写明是否希望该项目以及相应的个更新被包含在 [Filecoin newsletter](https://mailchi.mp/filecoin.io/subscribe)里。

   - 如果一切顺利，你的PR会被批准并且合并入库，这时候你就可以进入申请的下一步。

3. 在[devgrant提交Issues](https://github.com/filecoin-project/devgrants/issues)申请Micro-grants的Issue；

   项目展示的PR合并以后，提交新的Issue的申请Microgrant，可以通过`Next Step Microgrant Application`的模板快速创建，然后根据提示填写项目相关信息。请尽量提供详细的信息，可以帮助审批者更好的了解项目并且批准。可以参考[Microgrant applicaiton Issue](https://github.com/filecoin-project/devgrants/issues/266)

##### 项目申请参考 - Miner.Report

1. [[Applicaion] Miner.Report](https://github.com/filecoin-project/community/discussions/208)
2. [Project showcase submission](https://github.com/filecoin-project/community/blob/master/projects/application/miner-report-en.md)
3. [Microgrant applicaiton Issue](https://github.com/filecoin-project/devgrants/issues/266)

**Comment by zhaobo443** [comment]
FIL联合封装，用户提供质押币，公司提供设备和运维，收益60％,公司扣出20％服务费和20%设备费，合同期1080天，540天质押币退还，t3天封装。可实地考察。V:zhaobo9308

**Reply by beck-8** [reply]
我是真的烦你

**Comment by RichByDMC** [comment]
挖FIL的伙伴们看过来，一个火爆全网的全新存储项目自带公链钱包NFT等完整生态DMC，唯一能够超过Fil的项目诞生了。主要还能双挖FIL和DMC~~有想法的可以V我：dmctorich

**Reply by beck-8** [reply]
我也烦你
